src/components/app/details/AppHeader.tsx

```diff
@@ -84,8 +87,13 @@ export const AppHeader = ({
         currentPathname.current = location.pathname
     }, [location.pathname])
 
+    useEffect(() => {
+        setCurrentAppName(appName)
```

Comment:
Please re verify if it is required

Reply:
Verified and for safety added appName in dependency